drivers/media/i2c/arducam-pivariety.c

```diff
@@ -1357,9 +1357,8 @@ static int pivariety_probe(struct i2c_client *client,
 
 	pivariety->xclk_freq = clk_get_rate(pivariety->xclk);
 	if (pivariety->xclk_freq != 24000000) {
-		dev_err(dev, "xclk frequency not supported: %d Hz\n",
+		dev_warn(dev, "xclk frequency not supported: %d Hz\n",
```

Comment:
If this warning is now gone, do we need to keep the diff?

Reply:
We need to keep it; this diff removes the error, and downgrades it to a warning.